src/cli/install.ps1

```diff
@@ -1,176 +1,220 @@
-    Write-Output "The command 'curl.exe $URL -o $ZipPath' exited with code ${LASTEXITCODE}`n"
-    exit 1
-  }
-  if (!(Test-Path $ZipPath)) {
-    Write-Output "Install Failed - could not download $URL"
-    Write-Output "The file '$ZipPath' does not exist. Did an antivirus delete it?`n"
-    exit 1
-  }
-
-  try {
-    $lastProgressPreference = $global:ProgressPreference
-    $global:ProgressPreference = 'SilentlyContinue';
-    Expand-Archive "$ZipPath" "$BunBin" -Force
-    $global:ProgressPreference = $lastProgressPreference
-    if (!(Test-Path "${BunBin}\$Target\bun.exe")) {
-      throw "The file '${BunBin}\$Target\bun.exe' does not exist. Download is corrupt / Antivirus intercepted?`n"
-    }
-  } catch {
-    Write-Output "Install Failed - could not unzip $ZipPath"
-    Write-Error $_
-    exit 1
-  }
-  Move-Item "${BunBin}\$Target\bun.exe" "${BunBin}\bun.exe" -Force
-
-  Remove-Item "${BunBin}\$Target" -Recurse -Force
-  Remove-Item $ZipPath -Force
-
-  $BunRevision = "$(& "${BunBin}\bun.exe" --revision)"
-  if ($LASTEXITCODE -eq 1073741795) { # STATUS_ILLEGAL_INSTRUCTION
-    if ($IsBaseline) {
-      Write-Output "Install Failed - bun.exe (baseline) is not compatible with your CPU.`n"
-      Write-Output "Please open a GitHub issue with your CPU model:`nhttps://github.com/oven-sh/bun/issues/new/choose`n"
-      exit 1
-    }
+<#
+.SYNOPSIS
 
-    Write-Output "Install Failed - bun.exe is not compatible with your CPU. This should have been detected before downloading.`n"
-    Write-Output "Attempting to download bun.exe (baseline) instead.`n"
-
-    Install-Bun -Version $Version -ForceBaseline $True
-    exit 1
-  }
-  if ($LASTEXITCODE -eq 3221225781) { # STATUS_DLL_NOT_FOUND
-    Write-Output "Install Failed - You are missing a DLL required to run bun.exe"
-    Write-Output "The command '${BunBin}\bun.exe --revision' exited with code ${LASTEXITCODE}`n"
-    exit 1
-  }
-  if ($LASTEXITCODE -ne 0) {
-    Write-Output "Install Failed - could not verify bun.exe"
-    Write-Output "The command '${BunBin}\bun.exe --revision' exited with code ${LASTEXITCODE}`n"
-    exit 1
-  }
-  $DisplayVersion = if ($BunRevision -like "*-canary.*") {
-    "${BunRevision}"
-  } else {
-    "$(& "${BunBin}\bun.exe" --version)"
-  }
-
-  $C_RESET = [char]27 + "[0m"
-  $C_GREEN = [char]27 + "[1;32m"
-
-  try {
-    $null = New-Item -ItemType HardLink -Path "${BunBin}\bunx.exe" -Target "${BunBin}\bun.exe" -Force
-  } catch {
-    Write-Warning "Could not create a hard link for bunx, falling back to a cmd script`n"
-    Set-Content -Path "${BunBin}\bunx.cmd" -Value "@%~dp0bun.exe x %*"
-  }
+A script to install bun.
+
+.DESCRIPTION
+
+This script installs bun. By default, the installation directory is %LOCALAPPDATA%\Programs\bun.
+The installation directory can be customized by specifying the custom installation path through the %BUN_INSTALL% environment variable.
+To install bun to %USERPROFILE%\.bun, simply use the -InstallToUserProfile switch.
```

Comment:
currently the code refers to `%USERPROFILE%\.bun` for where to put data like installation cache and other things. This matches how bun behaves on linux and MacOS.

if this is to be changed in the installation script, it must also be changed in the code so that package management is all in the same `$Env:BUN_INSTALL`